Doc/c-api/intro.rst

```diff
@@ -105,6 +105,12 @@ defined closer to where they are useful (e.g. :c:macro:`Py_RETURN_NONE`).
 Others of a more general utility are defined here.  This is not necessarily a
 complete listing.
 
+.. c:macro:: PyMODINIT_FUNC
+
+   Declare an extension module ``PyInit`` initialization function. The return
+   type is :c:expr:`PyObject*`. The macro should only be used in the Python C
+   API.
```

Comment:
"The macro should only be used in the Python C API." -- I think that it is wrong.

This description is not clear, and does not help. The use of `PyMODINIT_FUNC` is actually described in `Doc/extending/extending.rst`:

```rst
This structure, in turn, must be passed to the interpreter in the module's
initialization function.  The initialization function must be named
:c:func:`!PyInit_name`, where *name* is the name of the module, and should be the
only non-\ ``static`` item defined in the module file::

   PyMODINIT_FUNC
   PyInit_spam(void)
   {
       return PyModule_Create(&spammodule);
   }

Note that :c:macro:`PyMODINIT_FUNC` declares the function as ``PyObject *`` return type,
declares any special linkage declarations required by the platform, and for C++
declares the function as ``extern "C"``.
```

It would be better if PyMODINIT_FUNC links would refer to that text. Or at least make a reference to it from the declaration in `Doc/c-api/intro.rst`.

Reply:
Ok, I copied this doc to Doc/c-api/intro.rst.